ee/debug/checkups/services_windows.go

```diff
@@ -228,15 +232,46 @@ func gatherServices(z *zip.Writer, serviceManager *mgr.Mgr) error {
 	return nil
 }
 
+// gatherServiceManagerEvents uses Get-WinEvent to fetch the service manager logs. This might be newer than Get-EventLog
+func gatherServiceManagerEvents(ctx context.Context, z *zip.Writer) error {
+	out, err := z.Create("eventlog-Get-WinEvent.json")
+	if err != nil {
+		return fmt.Errorf("creating eventlog-Get-WinEvent.json: %w", err)
+	}
+
+	filterExpression := fmt.Sprintf(`'@{LogName='System'; ProviderName='Service Control Manager'; Data=%s}'`, kolideSvcName)
+
+	cmdArgs := []string{
+		"Get-WinEvent",
+		"-MaxEvents 100",
+		"-FilterHashtable", filterExpression,
+		"|",
+		"ConvertTo-Json",
+	}
+
+	cmd, err := allowedcmd.Powershell(ctx, cmdArgs...)
+	if err != nil {
+		return fmt.Errorf("creating powershell command: %w", err)
+	}
+	hideWindow(cmd)
+	cmd.Stdout = out // write directly to zip
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("running Get-WinEvent: error %w", err)
+	}
```

Comment:
```suggestion
	cmd.Stdout = out // write directly to zip
	errBytes := bytes.Buffer{}
	cmd.Stderr = &errBytes
	if err := cmd.Run(); err != nil {
		return fmt.Errorf("running Get-WinEvent: error %w; stderr %s", err, errBytes.String())
	}
```

When testing locally I remembered that Windows stdout doesn't contain useful error info and we have to capture stderr if we want to get the actual error message.

(This suggestion isn't standalone because it also requires adding a "bytes" import to this file -- if it's easier I can add this here and to the `Get-EventLog` cmd below in a follow-up PR.)

Reply:
As a potential counter proposal, what do you think about `cmd.Stderr = out` ?

Reply:
Works for me also!

Reply:
Did that. If we discover issues, we can always change.